pkg/kubelet/dockertools/docker.go

```diff
@@ -45,6 +45,7 @@ import (
 const (
 	PodInfraContainerName = leaky.PodInfraContainerName
 	DockerPrefix          = "docker://"
+	DockerPullablePrefix  = "docker-pullable://"
```

Comment:
It lets us tell which value (config digest vs manifest digest/spec) we're getting at a glance.  Otherwise, there's not an easy way to tell, except by trying to parse and/or inspect the value